### PR TITLE
add source-definitions

### PR DESCRIPTION
**Pull request by vovavovavovavova**
## What
Added source_definitions to airbyte config for surveymonkey source

### CODE REVIEW

File: airbyte-config/init/src/main/resources/seed/source_definitions.yaml [hunk 1]
```diff
@@ -348,3 +348,8 @@
   dockerRepository: airbyte/source-okta
   dockerImageTag: 0.1.2
   documentationUrl: https://docs.airbyte.io/integrations/sources/okta
+- sourceDefinitionId: badc5925-0485-42be-8caa-b34096cb71b5
+  name: Survey Monkey
+  dockerRepository: airbyte/source-surveymonkey
+  dockerImageTag: 0.1.0
+  documentationUrl: https://hub.docker.com/r/airbyte/source-surveymonkey
```
Comment (keu) [inline airbyte-config/init/src/main/resources/seed/source_definitions.yaml:355]: use link to docs

File: airbyte-config/init/src/main/resources/config/STANDARD_SOURCE_DEFINITION/badc5925-0485-42be-8caa-b34096cb71b5.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "sourceDefinitionId": "badc5925-0485-42be-8caa-b34096cb71b5",
+  "name": "Survey Monkey",
+  "dockerRepository": "airbyte/source-surveymonkey",
+  "dockerImageTag": "0.1.0",
+  "documentationUrl": "https://hub.docker.com/r/airbyte/source-surveymonkey"
```
Comment (keu) [inline airbyte-config/init/src/main/resources/config/STANDARD_SOURCE_DEFINITION/badc5925-0485-42be-8caa-b34096cb71b5.json:6]: use link to docs

**Review by keu — CHANGES_REQUESTED** [review]
you didn't follow the connector checklist.
Please ensure you have added docs, updated summary, status page, etc

File: docs/integrations/sources/surveymonkey.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+# PostHog
```
Comment (sherifnada) [inline docs/integrations/sources/surveymonkey.md:1]: Surveymonkey

File: airbyte-integrations/connectors/source-surveymonkey/CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# Changelog
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-surveymonkey/CHANGELOG.md:1]: this file is no longer needed

File: docs/integrations/sources/surveymonkey.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+| Feature | Supported?\(Yes/No\) | Notes |
+| :--- | :--- | :--- |
+| Full Refresh Sync | Yes |  |
+| Incremental Sync | Yes |  |
+| Namespaces | No |  |
+
+### Performance considerations
+
+The SurveyMonkey API have request limitation. Your default private app has the following limits:
```
Comment (sherifnada) [inline docs/integrations/sources/surveymonkey.md:36]:
```suggestion
The SurveyMonkey API applies heavy API quotas for default private apps, which have the following limits:
```

**Review by sherifnada — APPROVED** [review]

**Comment by sherifnada** [comment]
@vovavovavovavova please also add it to  `airbyte-integrations/connectors/README.md`

File: docs/integrations/sources/surveymonkey.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+### Requirements
+
+* SurveyMonkey API Key
+
+### Setup guide
+
+Please read this [docs](https://developer.surveymonkey.com/api/v3/#getting-started).
+Register your application [here](https://developer.surveymonkey.com/apps/)
+Then go to Setting and copy your access token
```
Comment (keu) [inline docs/integrations/sources/surveymonkey.md:54]:
```suggestion
Then go to Settings and copy your access token 
```

**Review by keu — APPROVED** [review]

**Pull request merged**